Add RNASeq and Sarek delivery readmes (#57)
SciLifeLab/taca-ngi-pipeline: delivery_readmes/DELIVERY.README.RNASeq.txt

README
=======

The README describes the content of the delivered data.

> -----------------------------------------------------------------

Reviewer: Consistency in line lengths (either that or match the length of the header)

```suggestion
--------------------------------------------------------------------
```

> The README describes the content of the delivered data.
> -----------------------------------------------------------------

Reviewer: In fact I'm not really sure what this header is saying? Maybe remove the line under it and have an introduction sentence...

```suggestion
Many thanks for sequencing your samples with the SciLifeLab National Genomics Infrastructure!
This README describes the different files present in your data delivery, including information about verifying file integrity and finding results.
If you have any questions, please don't hesitate to get in touch: support@ngisweden.se
```

====================================================================
ProjectID
====================================================================
The root folder, which is named by the NGI project ID, contains one report folder, one analysis results folder and a folder for each sample.

> Each sample folder is accompanied by a .lst-file containing a list of the files in the folder and a .md5-file containing the MD5-checksums of the files in the folder.

Reviewer: ```suggestion
Each sample folder is accompanied by a .lst file containing a list of the files in the folder and a .md5 file containing the MD5-checksums of the files in the folder.
```

> Use the MD5-checksums to verify the integrity of the files after transfer.

Reviewer: Consistent capitalisation, un-needed-hyphenation 😄 

```suggestion
Use the md5 checksums to verify the integrity of the files after transfer.
```

├──ProjectID
	├── 00-Reports
	├── 01-RNA-Results

>         ├── Sample1

Reviewer: Inconsistent indentation

Author: Fixed now hopefully!

        ├── Sample1.lst
        ├── Sample1.md5

>         ├── Sample2

Reviewer: If a folder, good to suffix with `/` so that it's clear that this isn't a flat file like the rest.

Author: Done!

        ├── Sample2.lst
        ├── Sample2.md5
	...
	...
        ├── SampleN
        ├── SampleN.lst
        └── SampleN.md5

The root folder contains following files as well:

> ├──ProjectID

Reviewer: Is there a reason not to include these in the same file tree above?

        ├── ACKNOWLEDGEMENTS.txt

>         ├── DELIVERY.README.SAREK.txt

Reviewer: This is rnaseq

Author: Updated!

        ├── miscellaneous.lst
        └── miscellaneous.md5

--ACKNOWLEDGEMENTS.txt
A file describing what to include in any publications using this data.

--miscellaneous.lst
A list of the files of the delivery, excluding raw data.

--miscellaneous.md5
A file to check the integrity of the miscellaneous.lst file.

File integrity checks
---------------------

> It is extremely important to check the integrity of the delivered data using the .md5 files. To do this, run the following:
> md5sum -c file_with_checksums.md5
> For more information, please see https://ngisweden.scilifelab.se/resources/data-delivery/

Reviewer: Whitespace around commands, make it clearer that the command contains an example string that should be replaced.

```suggestion
It is extremely important to check the integrity of the delivered data using the .md5 files. To do this, run the following command in the terminal (replace with the relevant file name):

md5sum -c [checksum_file.md5]

For more information, please see https://ngisweden.scilifelab.se/resources/data-delivery/
```

====================================================================
ProjectID -> 00-Reports
====================================================================
00-Reports folder contains the reports with the sample and sequence QC. It also contains a project summary report and MultiQC reports.

├── 00-Reports
	├── ProjectName_project_summary.html
	├── ProjectName_sample_info.txt
	├── ProjectName_lane_info.txt
	├── ProjectName_library_info.txt
	├── ProjectName_multiqc_report.html
	└── manifestFiles
		└── <ProjectID>.02-FASTQ.<flowcell>.<ProjectID>_<BCL-CONVERSION-ID>_<lane-number>_manifest.txt

--ProjectName_project_summary.html

> A report summarizing information about the project

Reviewer: ```suggestion
A report summarizing information about the project.
```

--ProjectName_[sample/lane/library]_info.txt

> Tab-separated text files for each table found in the report, convenient for loading into Excel

Reviewer: ```suggestion
Tab-separated text files for each table found in the report, convenient for loading into Excel.
```

--ProjectName_multiqc_report.html
Report summarizing general quality statistics based on the bioinformatics tools FastQC and FastQ Screen.

--<ProjectID>.02-FASTQ.<flowcell>.<ProjectID>_<BCL-CONVERSION-ID>_<lane-number>_manifest.txt
A file summarizing the run with information about instruments and library.

====================================================================
ProjectID -> 01-RNA-Results
====================================================================
Contains best practics analysis results from the nf-core/rnaseq pipeline. Refer to

> https://github.com/nf-core/rnaseq/blob/1.4.2/docs/output.md for output description.

Reviewer: Do we want to tie this readme to a specific version of the pipeline? We’ll need to be very careful about updating it if so..

Author: Jun said we will change this to https://nf-co.re/rnaseq/output and that she will include text about choosing the right version in the email she sends to the user.

Reviewer: Perfect 👌🏻

├── 01-RNA-Results
	├── dupradar
	├── fastqc

> 	├── featureCounts

Reviewer: As with the pipeline version, this will change with different versions of the pipeline. We won't run featureCounts with v3.0 of the RNA-seq pipeline for example. I think it's probably better to just keep the top-level folder name.

Author: Removed the individual tool result directories.
